util/env.go

```diff
@@ -0,0 +1,51 @@
+package util
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"golang.org/x/term"
+)
+
+func Getenv(key, fallback string) string {
+	if value, ok := os.LookupEnv(key); ok {
+		return value
+	}
+	return fallback
+}
+
+func GetenvBool(key string) bool {
+	return strings.ToLower(Getenv(key, "false")) == "true"
+}
+
+func GetenvPrompt(key string, prompt ...string) string {
+	value := Getenv(key, "")
+	if value == "" {
```

Comment:
Should we use a `while value == ""` ? To prevent mistakes.

Reply:
It makes sense to me, nice addition :)

```suggestion
	while value == "" {
```